servicex/transformer/arrow_writer.py

```diff
@@ -60,6 +62,7 @@ def write_branches_to_arrow(self, transformer,
 
             for batch in batches:
                 if self.messaging:
```

Comment:
This might not be related to the current PR, but should lines 63 and 64 be switched so we don't check if Kafka is enabled separately for each batch? I.e.

            if self.messaging:
                for batch in batches:

Reply:
Oh, nice catch.
Added a new commit to fix this and also took this opportunity to reduce the logging inside this batches loop. It generates a bunch of noise and can slow down Kafka publication for a large file